### DISCUSSION TITLE
github processing update takes very long

### DISCUSSION THREAD

**Discussion by MarenOelixtown** [neuefische/questions#850] [Web] [answered]
Hi there, after pushing a commit to github, this commit is not reflecting in the commit-overview of the featcher-branch. 'Processing updates' are working about 20min now. Belongs this issue to the github-server or can we fix it from our side? Thanks in beforehand...

**Comment by MarenOelixtown** [comment, marked as answer]
Hi Nikas, just a second ago the process is completet!🫡

**Reply by NikRoe** [reply]
Awesome! Then it was just a delay on GitHub's side, I assume.
